modules/claims_api/app/services/claims_api/slack/failed_submissions_messenger.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+
+module ClaimsApi
+  module Slack
+    class FailedSubmissionsMessenger
+      # rubocop:disable Metrics/ParameterLists
+      def initialize(claims, poa, itf, ews, from, to)
+        @errored_claims = claims
+        @errored_poa = poa
+        @errored_itf = itf
+        @errored_ews = ews
+        @to = to
+        @from = from
+      end
+      # rubocop:enable Metrics/ParameterLists
+
+      def notify!
+        slack_client = SlackNotify::Client.new(webhook_url: Settings.claims_api.slack.webhook_url,
+                                               channel: '#api-benefits-claims-alerts',
+                                               username: 'Failed Submissions Messenger')
+
+        notification_message = build_notification_message
+
+        slack_client.notify(notification_message)
+      end
+
+      private
+
+      def build_notification_message
+        message = ''.dup
+        message << message_heading
+        message << build_submission_information(@errored_claims, 'Disability Compensation')
+        message << build_submission_information(@errored_poa, 'Power of Attorney')
+        message << build_submission_information(@errored_itf, 'Intent to File')
+        message << build_submission_information(@errored_ews, 'Evidence Waiver')
+        message
+      end
+
+      def message_heading
+        heading = ''.dup
+        heading << build_heading_message
+        heading
+      end
+
+      def build_heading_message
+        heading_message = ''.dup
+        heading_title = "*ERRORED SUBMISSIONS* \n\n"
+        heading_message << heading_title
+        heading_timeframe = "#{@from} - #{@to} \n"
+        heading_message << heading_timeframe
+        heading_sub_text = "The following submissions have encountered errors in *#{@environment}*. \n\n"
+        heading_message << heading_sub_text
```

Comment:
do we need the duplication in each of these? what's ur thoughts on combining each of these lines? 
```
heading_message << "*ERRORED SUBMISSIONS* \n\n"
heading_message << "#{@from} - #{@to} \n"
heading_message << "The following submissions have encountered errors in *#{@environment}*. \n\n"